### ISSUE TITLE
machine.py tools vs tool attribute distinction

### ISSUE THREAD

**Issue by brendenpelkie** [machineagency/science_jubilee#6]
the Machine object has both a 'tool' (the current tool) attribute as well as 'tools' (the list of available tools). The distinction between these should be made more clear by renaming one.

**Comment by bsubbaraman** [comment]
Good point (and there's also `configured_tools` which is reads from the RRF object model!). I propose changing `tools` to `loaded_tools` (since this information is updated after a `load_tool()` command. Maybe it's worth also renaming `tool` to `active_tool` to be explicit?
